src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/PublicationService.cs

```diff
@@ -25,6 +25,12 @@ public PublicationService(ContentDbContext context, IMapper mapper, IReleaseServ
             _releaseService = releaseService;
         }
 
+        public async Task<Publication> Get(Guid id)
+        {
+            return await _context.Publications
+                .SingleOrDefaultAsync(publication => publication.Id == id);
```

Comment:
I've found using `FindAsync(id)` is more succinct, although you might need to pop the `Key` annotation onto `Publication.Id`.